csharp/ql/test/library-tests/definitions/PrintAst.expected

```diff
@@ -175,10 +175,10 @@ definitions.cs:
 #   97|   5: [Class] A
 #   99|     5: [DelegateType] EventHandler
 #  101|     6: [Event] Click
-#  101|       3: [RemoveEventAccessor] remove_Click
+#  101|       3: [AddEventAccessor] add_Click
```

Comment:
I'm not sure what controls the order of these.

Reply:
I think it is the order in which they are extracted.

Reply:
Does this mean that the order is not stable, and we'll see the order of these accessors swap?

Reply:
I would expect it to be stable for a given version of the extractor. Looking at the code in `definitions.cs`, these are in fact compiler generated, so I guess they should be hidden in the AST viewer?

Reply:
I decided to include all accessors regardless of them being compiler generated or not. I think it makes sense, especially for autoproperties. https://github.com/github/codeql/blob/main/csharp/ql/src/semmle/code/csharp/PrintAst.qll#L39

Reply:
In that case we could decide in `PrintAst.qll` that `get` always comes before `set` and `add` before `remove` for auto-generated accessors.